src/cmd/src/cli.rs

```diff
@@ -22,7 +22,7 @@ mod helper;
 
 // Wait for https://github.com/GreptimeTeam/greptimedb/issues/2373
 #[allow(unused)]
-mod repl;
```

Comment:
`repl` module use `Database` but its not used right now, so I disable it.

Reply:
We should have an EXPLICIT todo or create an issue for it. This looks quite dirty and easy to create debt ignored in a long time.

Reply:
There is a `repl` related issue #2373 already, mentioned at https://github.com/GreptimeTeam/greptimedb/blob/e6eca8ca0c56247d278ca049c98796456699c6c6/src/cmd/src/cli.rs#L128

Maybe leave a `TODO` here?

Reply:
Fair enough. I just want to ensure that all this half-complete code can converge instead of leaving unused code in our codebase for _years_.